### PR TITLE
chore(gitlab): update gitlab chart - bump container image version, change redis chart repo, bump chart version

### PR DESCRIPTION
**Pull request by seanrmurphy**
There was an issue with v0.6.0 of this helm chart which impacted how prometheus was scraping data from gitlab; that is the primary driver for the new release of this chart (#2656). 

It was also noticed that the dependent redis chart must be changed due to the issues with bitnami changing how they publish artifacts and also the gitlab container image was a bit out of date.

This has been deployed within a dedicated namespace, with the renku gitlab dep pointing at this local version 0.6.1 of the gitlab helm chart.

### CODE REVIEW

File: chartpress-gitlab.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# this file should be used as and when necessary to build the renku gitlab helm
+# chart. As renku moves to use of the standard gitlab cloud native chart, this
+# will become redundnat but for now it is still in use.
+#
+# To use this, you will need to remove the chartpress.yaml for building renku and
```
Comment (rokroskar) [inline chartpress-gitlab.yaml:5]: this seems sub-optimal - can we contribute a `--file` option to the upstream?

Comment (seanrmurphy) [reply chartpress-gitlab.yaml:5]: Sounds v doable - prob just adding an arg with a default and prob a test.

I can look into it.

**Review by pameladelgado — COMMENTED** [review]
It looks good to me, but I kind of prefer also to have the commented section in the chartpress file.

File: chartpress-gitlab.yaml (hunk 1 above)
Comment (rokroskar) [reply chartpress-gitlab.yaml:5]: I made https://github.com/jupyterhub/chartpress/issues/170

**Comment by rokroskar** [comment]
what's the status here @seanrmurphy ?

**Comment by seanrmurphy** [comment]
Closing - superceded by #2820.

**Pull request closed without merging**